docker/1.4.4/README.md

```diff
@@ -13,9 +13,9 @@ Three mount points have been created in the image to be used for configuration,
 ##Configuration
 
 When running the image, the default configuration values are used. 
-To use a custom configuration file, mount a **local** configuration file to `/mosquitto/conf/mosquitto.conf`
+To use a custom configuration file, mount a **local** configuration file to `/mosquitto/config/mosquitto.conf`
 ```
-docker run -it -p 1883:1883 -p 9001:9001 -v mosquitto.conf:/mosquitto/data/mosquitto.conf mosquitto:1.4.4
+docker run -it -p 1883:1883 -p 9001:9001 -v `pwd`/mosquitto.conf:/mosquitto/config/mosquitto.conf mosquitto:1.4.4
```

Comment:
Is the `pwd` necessary. It is just an example where the configuration file is in the current path. If this is confusing we could just use `<path-to-configuration-file>` instead maybe?